### PR TITLE
WAIT FOR RELEASE 46581 add install order docs

### PR DESCRIPTION
**Pull request by paigecalvert**
Helm install order how-to doc: https://deploy-preview-332--replicated-docs.netlify.app/vendor/helm-native-helm-install-order

HelmChart reference: https://deploy-preview-332--replicated-docs.netlify.app/reference/custom-resource-helmchart#weight

https://app.shortcut.com/replicated/story/46581/blue-team-document-helm-chart-install-order

### CODE REVIEW

**Comment by JustinN-Replicated** [comment]
Just a quick question, but do we need to be more explicit about the order - in other words, KOTS will run the 'helm install' command for the lowest chart weight first, wait until it has completed installing in helm, and then run the next 'helm install' command for the next lowest chart weight etc. I think this is implied, but wanted to mention to see others' opinions. This may go too granular - I could go either way but it is not a blocker for this docs pr

**Review by JustinN-Replicated — APPROVED** [review]
lgtm; comments I added are not critical or blocking

**Comment by paigecalvert** [comment]
> Just a quick question, but do we need to be more explicit about the order - in other words, KOTS will run the 'helm install' command for the lowest chart weight first, wait until it has completed installing in helm, and then run the next 'helm install' command for the next lowest chart weight etc. I think this is implied, but wanted to mention to see others' opinions. This may go too granular - I could go either way but it is not a blocker for this docs pr

Linking for our future selves the conversation we had around this question in slack: https://replicated.slack.com/archives/C02TC9ZN519/p1651174281650599

File: docs/vendor/helm-native-helm-install-order.md [hunk 1]
```diff
@@ -1,8 +1,75 @@
...
+1. Open the manifest file for the `HelmChart` custom resource you want to assign an installation order.
+
+1. Add the following line to the `HelmChart` custom resource manifest file at the same level as `useHelmInstall: true`:
+
+   ```yaml
+   weight: INSTALL-WEIGHT
+   ```
+   Where `INSTALL-WEIGHT` is any negative or positive integer or `0`.
+
+   **For example**:
```
Comment (jonquil2002) [inline docs/vendor/helm-native-helm-install-order.md:34]: So, I've been using **Example** or **Example:** instead. We just need to add this to our discussion list for the Style Guide.

File: docs/vendor/helm-native-helm-install-order.md [hunk 2]
```diff
@@ -1,8 +1,75 @@
...
+     values:
+       ...
+
+     namespace: samplechart-namespace
+   ```
+1. Save your changes.
+
+## Subcharts and Dependencies
+
+When you add a `weight` property to `HelmChart` custom resources in your application, The app manager instructs Helm to install any dependencies, including subcharts, along with the parent chart.
```
Comment (jonquil2002) [inline docs/vendor/helm-native-helm-install-order.md:62]: s/ The app manager / the app manager

"The" should be "the"

Comment (JustinN-Replicated) [reply docs/vendor/helm-native-helm-install-order.md]: changed

File: docs/vendor/helm-native-helm-install-order.md (hunk 1 above)
Comment (JustinN-Replicated) [reply docs/vendor/helm-native-helm-install-order.md:34]: Going to change to "Example:"

Comment (JustinN-Replicated) [reply docs/vendor/helm-native-helm-install-order.md]: changed

File: docs/vendor/helm-native-helm-install-order.md [hunk 3]
```diff
@@ -1,8 +1,75 @@
...
+
+For example, if you have two native Helm charts in your application, one with a `weight` of `-1` and one with a weight of `0`, then Helm installs the chart with a `weight` of `-1` first, including any subcharts and dependencies listed in the `dependencies` field for that chart.
```
Comment (jonquil2002) [inline docs/vendor/helm-native-helm-install-order.md:64]: s/ weight of `0` / `weight` of `0`

**Review by JustinN-Replicated — APPROVED** [review]
lgtm not sure why this was on hold

**Pull request merged**